packages/nodes-base/nodes/SendInBlue/SenderDescrition.ts

```diff
@@ -0,0 +1,196 @@
+import { IExecuteSingleFunctions, INodeExecutionData, INodeProperties } from 'n8n-workflow';
+
+export const senderOperations: INodeProperties[] = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: ['sender'],
+			},
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				action: 'Create a sender',
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				routing: {
+					request: {
+						method: 'DELETE',
+						url: '=/v3/senders/{{$parameter.id}}',
+					},
+					output: {
+						postReceive: [
+							{
+								type: 'set',
+								properties: {
+									value: '={{ { "success": true } }}',
+								},
+							},
+						],
+					},
+				},
+				action: 'Delete a sender',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				routing: {
+					request: {
+						method: 'GET',
+						url: '/v3/senders',
+					},
+					send: {
+						paginate: false,
+					},
+					output: {
+						postReceive: [
+							{
+								type: 'rootProperty',
+								properties: {
+									property: 'senders',
+								},
+							},
+							async function (
+								this: IExecuteSingleFunctions,
+								items: INodeExecutionData[],
+							): Promise<INodeExecutionData[]> {
+								const returnAll = this.getNodeParameter('returnAll') as boolean;
```

Comment:
Isn't it possible to send this as a request param to fetch the exact number already? If not, you can disregard this comment.

Reply:
No, they actually don't support query parameters on that api call

[getSenders](https://developers.sendinblue.com/reference/getsenders-1)